sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -51,12 +50,18 @@ import { ClientTypeBaseRoute } from "./types";
 import { AppStudentsModule } from "./app.students.module";
 import { AppSystemAccessModule } from "./app.system-access.module";
 import { AppSupportingUsersModule } from "./app.supporting-users.module";
+import {
+  ZeebeModule,
+  SequenceControlService,
+  WorkflowClientService,
+} from "@sims/services";
 
 @Module({
   imports: [
     LoggerModule,
     DatabaseModule,
     AuthModule,
+    ZeebeModule.forRoot(),
```

Comment:
mm.. just curious what is `.forRoot(),`

Reply:
Usually, the modules are registered `forRoot` or `forFeature`. For Zeebe it is registered with `global: true` which means that it will be globally available for injection. I tried to follow the naming convention below. We also do similar for DB `TypeOrmModule.forRoot`.

![image](https://user-images.githubusercontent.com/61259237/196803033-9d3c1194-be17-4636-bd2a-9dbb129a787e.png)
Source: https://docs.nestjs.com/modules